src/main/scala/PropertyChecker.scala

```diff
@@ -243,7 +253,7 @@ object PropertyChecker {
       throw new MetadataError("common property with @language lacks a @value")
     }
     val matcher = Bcp47Language.r.pattern.matcher(v.asText())
-    if (!matcher.matches() && !v.isEmpty) {
+    if (!matcher.matches() || v.isEmpty) {
```

Comment:
Is this `!(matcher.matches() || v.isEmpty)` or `(!matcher.matches()) || v.isEmpty`?

Reply:
If the language value does not match a BCP47 Language or if the language value is empty execute the code in if block.

Isn't this `(!matcher.matches()) || v.isEmpty`  same as what is already there?